app/views/controllers/shared/_form_ccby40.erb

```diff
@@ -0,0 +1,9 @@
+<%=
+tag.div(id: "license") do
+  link_to("https://creativecommons.org/licenses/by/4.0/", rel: "license") do
+    image_tag("https://i.creativecommons.org/l/by/4.0/88x31.png",
+              alt: "Creative Commons Attribution License",
+              class: "border-none")
+  end
+end
+%>
```

Comment:
Template for license badge. 
This and the following templates anticipate adding these licenses to the db.

Reply:
It seems odd to me that there are partials for each license if we are expecting admins to add licenses   Perhaps the license object can be parameterized with the CC.org links so there can be a single generic partial.

Reply:
Thanks for the suggestion. I'll investigate way(s) of having a generic partial.

Reply:
@mo-nathan This turns out to be very easy; CC has changed urls so that we don't even have to parametize the license object. In fact, I'm going to ditch the partial and put the code in the show template.